interface/forms/eye_mag/js/eye_base.php

```diff
@@ -1032,13 +1032,12 @@ function editScripts(url) {
         let title = 'Prescriptions';
         let w = 810;
         w = 910;
+        function refreshme() {
+            top.restoreSession();
+            location.reload();
+        }
 
-        dlgopen(url, 'editScripts', w, 300, '', '', {
-            buttons: [
-                {text: 'Add', close: false, style: 'primary  btn-sm', click: AddScript},
-                {text: 'List', close: false, style: 'primary  btn-sm', click: ListScripts},
-                {text: 'Done', close: true, style: 'default btn-sm'}
-            ],
+        dlgopen(url, 'editScripts', w, 400, '', '', {
             onClosed: 'refreshme',
             allowResize: true,
             allowDrag: true,
```

Comment:
or since this I've updated dialog to support promise so
dlgopen(url, 'editScripts', w, 400, '', '', { // this should autosize if over the specified height.
            resolvePromiseOn: 'close',
            allowResize: true,
            allowDrag: true,
            dialogId: 'editscripts',
            type: 'iframe' // default
        }).then(() => {
            top.restoreSession();
            location.reload();
            });

Much cleaner imo plus nobody seems to realize this promise opportunity using dialog and promise can be set to any of the BS modal events. Just saying and hawking all the work I've put into dialog!:)

Reply:
Sounds good, made the change.

Reply:
Cool Thanks! Provides another demo opportunity for new devs to use.